gnd/src/test/java/com/google/android/gnd/MainViewModelTest.java

```diff
@@ -146,4 +146,19 @@ public void testSignInStateChanged_onSignInError() {
     Mockito.verify(mockUserRepository, times(1)).clearUserPreferences();
     Mockito.verify(mockTosRepository, times(1)).setTermsOfServiceAccepted(false);
   }
+
+  @Test
+  public void testSignInStateChanged_onSignedIn_whenTosMissing() {
+    when(mockTosRepository.isTermsOfServiceAccepted()).thenReturn(false);
+    when(mockUserRepository.saveUser(any(User.class))).thenReturn(Completable.complete());
+    when(mockTosRepository.getTermsOfService()).thenReturn(Maybe.empty());
+
+    fakeAuthenticationManager.setUser(TEST_USER);
+    fakeAuthenticationManager.signIn();
+
+    assertProgressDialogVisible(false);
+    assertNavigate(HomeScreenFragmentDirections.showHomeScreen());
+    Mockito.verify(mockUserRepository, times(1)).saveUser(TEST_USER);
```

Comment:
Nit: Should we static import these Mockito methods as well for consistency?

Reply:
I've fixed this class in https://github.com/google/ground-android/pull/1031. Let's merge that first in order to fix here as well

Reply:
Good idea. Merged, reran CI.